libraries/botbuilder-dialogs-adaptive-teams/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "botbuilder-dialogs-adaptive-teams",
+  "author": "Microsoft Corp.",
+  "description": "Rule system for the Microsoft BotBuilder adaptive dialog system, with integration specific to Microsoft Teams.",
+  "version": "4.1.6",
```

Comment:
is 4.1.6 the version currently used in the rest of the sdk?

Reply:
Unfortunately, yes

Reply:
[yeah](https://github.com/microsoft/botbuilder-js/blob/main/libraries/botbuilder-core/package.json#L5)